melodic/distribution.yaml

```diff
@@ -2017,6 +2017,12 @@ repositories:
       url: https://github.com/clearpathrobotics/cpr_multimaster_tools.git
       version: kinetic-devel
     status: maintained
+  cpu_temperature_diagnostics:
+    source:
```

Comment:
could you also add a doc entry?

Reply:
Thank you for the suggestion. I added the doc entry.